### PR TITLE
mstats review

### PR DESCRIPTION
**Pull request by mhebing**
Updates based on the review of wp3 and wp4 (mstats)

### CODE REVIEW

File: instruments/models.py [hunk 1]
```diff
@@ -269,15 +272,24 @@ def to_topic_dict(self, language="en"):
...
             cs += ["%s: %s" % (a["value"], a["label"]) for a in item.get("answers", [])]
+        if wrap:
+            cs_temp = [ textwrap.wrap(line, wrap) for line in cs ]
+            cs = []
+            for wrap in cs_temp:
+                if wrap == []:
+                    cs.append("")
+                else:
+                    cs += wrap
+            #cs = [ line for wrap in cs2 for line in wrap ]
```
Comment (afuetterer) [inline instruments/models.py]: Please remove this line, if it is not used.

File: instruments/models.py [hunk 2]
```diff
@@ -269,15 +272,24 @@ def to_topic_dict(self, language="en"):
...
                 "",
-                "Item: %s (scale: %s)" % (item.get("value"), item.get("scale")),
+                "Item: %s (scale: %s)" % (item.get("item"), item.get("scale")),
                 item.get("text", ""),
             ]
             cs += ["%s: %s" % (a["value"], a["label"]) for a in item.get("answers", [])]
+        if wrap:
+            cs_temp = [ textwrap.wrap(line, wrap) for line in cs ]
+            cs = []
+            for wrap in cs_temp:
```
Comment (afuetterer) [inline instruments/models.py]: This "wrap" is not the same as the keyword argument of the method. It gets confusing if you say "if wrap" twice in a method, but the tested variable is actually a different thing. Can you change one of the variable names?

File: instruments/models.py (hunk 1 above)
Comment (mhebing) [reply instruments/models.py]: @afuetterer Done.

File: instruments/models.py (hunk 2 above)
Comment (mhebing) [reply instruments/models.py]: @afuetterer Done.

**Comment by afuetterer** [comment]
What's happening in https://github.com/ddionrails/ddionrails/blob/88822e82a03a87fed1055caf979cc288d8b61929/static/src/js/topic_list.js#L216 and https://github.com/ddionrails/ddionrails/blob/88822e82a03a87fed1055caf979cc288d8b61929/static/src/js/topic_list_extra_div.js#L153
?
Is the for loop doing anything?

**Comment by mhebing** [comment]
I think we can optimise the JS as soon as we have the debug working for JS/CSS, okay?

**Comment by afuetterer** [comment]
Try debugging JS with:

```shell
$ cd ddionrails/
$ pipenv shell
$ ./node_modules/.bin/webpack --config webpack.config.js --watch
```

**Pull request merged**